public/javascripts/index.js

```diff
@@ -41,17 +28,39 @@ const Letter = React.createClass({
 	    </div>
```

Comment:
Good question! I don't know where this came from but think I got rid of it when doing the latest bit of work.